### DISCUSSION TITLE
Follow day night in editor

### DISCUSSION THREAD

**Discussion by john-peterson** [massivemadness/Squircle-CE#242] [Ideas]
I want to use my own custom schemes instead of the pre built ones

**Comment by john-peterson** [comment]
https://github.com/john-peterson/Squircle-CE/commits/daynite/

**Reply by john-peterson** [reply]
This fails

How do I reference custom themes by name? Can I fetch the uuid? But settings need a string

~~~
settingsManager.colorScheme = "abc"

**Reply by john-peterson** [reply]
@massivemadness sorry to bother you with this question

**Reply by massivemadness** [reply]
@john-peterson the `colorScheme` in settings manager is an id, you can't use theme name. Every built-in theme has id (usually theme name in uppercase)
https://github.com/massivemadness/Squircle-CE/blob/f774f6936994b0145b12a7772820ae78e385f9a0/feature-themes/api/src/main/kotlin/com/blacksquircle/ui/feature/themes/domain/model/InternalTheme.kt#L24
